src/test/java/testsuite/integration/host/AuroraIntegrationContainerTest.java

```
@@ -255,6 +258,12 @@ public void testDebugPerformanceTestInContainer()
 
     container.start();
 
+    try {
+      container.execInContainer("dos2unix", "gradlew");
+    } catch (InterruptedException | UnsupportedOperationException | IOException e) {
+      return null;
```

Fail instead of returning null?